core/src/main/java/psiprobe/AbstractTomcatContainer.java

```diff
@@ -406,7 +406,7 @@ public void listContextJsps(Context context, Summary summary, boolean compile) {
   }
 
   @Override
-  public org.apache.juli.logging.Log getLogger(Context context) {
+  public Object getLogger(Context context) {
```

Comment:
Most likely a holdover from an older version of Tomcat. What you're saying makes sense. Let's remove it.